README.md

```diff
@@ -1,80 +1,190 @@
+* **[The full Meteor documentation](http://docs.meteor.com/#/full/quickstart)**
+* **[Atmospherejs.com - meteor packages](https://atmospherejs.com/)**
+* **[The iron router guide](http://iron-meteor.github.io/iron-router/)**
+* **[Collection FS github page (used for image upload)](https://github.com/CollectionFS/Meteor-CollectionFS)**
 
-[Meteor documentation](http://docs.meteor.com/#/full/)
+### Tasks
 
-## Tasks:
+####1. Structure
+This application is quite a lot bigger than the [chat application](https://github.com/EirikV/meteor-workshop/tree/chat-solution), so we have chosen to structure it into smaller files and folders. Structuring a larger Meteor app can be achieved using packages, but that falls outside the scope of this workshop. 
 
-### 1. First of all
+You should familiarize yourself with how the application is built. Poke a little around in the code, and don't hesitate to ask if some things are unclear!. 
 
-Head over to [instagram](https://instagram.com/developer/) and create an account if you don't have one already.
-Next, click "Register Your Application" and then "Register a New Client", to register a new client.
-Pop your `CLIENT ID` and `CLIENT SECRET` into **settings.json** (found in the root of the project).
+The application is built using the following structure:
 
-We do this because instagram has a 5000-requests an hour per application limit for applications where the user is not authenticated.
+* client (main folder for client side only code).
+    * common (holds common code such as global helpers)
+    * less
+    * modules (each logical piece of the application is treated like a module)
+        * some module
+            * some_root_level.html or .js
+            * some submodule folder
+                * some submodule .html or .js
+* lib (main folder for code shared between client and server).
+    * It may seem a little strange that the 'lib' folder doesn't actually contain any external libraries. We have named this folder lib as per Meteor's own [recommendations](http://docs.meteor.com/#/full/structuringyourapp) for structuring an app.
+* public (assets).
+* server (main folder for server side only code).
+    * modules (some of the client side modules also have server side counter parts. Others exist only on the server).
+        * some submodule
+            * some submodule .js
+#### 2. Routing
+Meteor applications mostly use one package for routing: the [iron:router](https://atmospherejs.com/iron/router). Using a router is essential for a larger application. With Meteor, using a router changes some of the basic mechanics of the framework. For example: In the chat application, you simply published a collection and then subscribed to it from the client. With a router, this needs to be handled for each route to make sure that a subscription is ready once a route function is executed. In the next series of subtasks, you will learn the parts of iron:router that are needed for Tweeteor.
```

Comment:
"you simply published a collection.." oh I did? 

Kanskje bedre å skrive "you can simply publish a collection"